web/modules/custom/joinup_notification/joinup_notification.module

```diff
@@ -30,18 +31,20 @@ function joinup_notification_entity_predelete(EntityInterface $entity) {
 
   $entity_type = $entity->getEntityTypeId();
   $bundle = $entity->bundle();
-  $field_definitions = array_filter(\Drupal::entityManager()->getFieldDefinitions($entity_type, $bundle), function ($field_definition) {
-    return $field_definition->getType() == 'state';
-  });
+  if ($entity_type !== 'rdf_entity' || $bundle !== 'asset_distribution') {
```

Comment:
Drop a line of documentation here to explain why only the asset_distribution bundle is excluded. I could understand it from the commit history, but let's make it easier for the next devs.